src/cli/commands/force/lightning/lwc/__tests__/preview.test.ts

```diff
@@ -117,7 +119,7 @@ describe('Preview Tests', () => {
         });
         jest.spyOn(CommonUtils, 'executeCommand').mockImplementation(cmdMock);
         const port = (await preview.isLwcServerRunning()).trim();
-        expect(port).toBe('3333');
+        expect(port.endsWith('3333')).toBeTrue();
```

Comment:
This test and the one below is failing for me. I'll check it out.

![image](https://user-images.githubusercontent.com/42476999/104532517-7c21db80-55b4-11eb-92bb-541ca45937e1.png)

Reply:
not sure why they're failing for you. on my machine they're passing and looks like CircleCI passed the build too.

Reply:
On my machine `port` in line 134 comes back as `undefined '3333'`. And this seems because of
![image](https://user-images.githubusercontent.com/42476999/104624284-79b49580-5637-11eb-8f92-28a984ae8f1c.png)
Still need to check what the delta is between your passing environment versus my failing environment. I got your branch so the code is exactly the same. I removed all node_modules and yarn'd so in theory there should be no variance.

Reply:
Don't waste your time on this. Following the PR feedback on the new approach, I'll be changing the test code anyway so I'll look into it.